trade-finance-manager-api/src/v1/rest-mappings/mappings/gef-facilities/mapGefFacility.test.js

```diff
@@ -65,4 +66,14 @@ describe('mapGefFacility', () => {
 
     expect(result).toEqual(expected);
   });
+
+  describe('when a tfm facilityStage exists', () => {
```

Comment:
Same as above RE combine into the main test above

Reply:
Done, thanks!